tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{27,34,35,36,py},py{27,36}-flake8,doclint,commitlint,docstrings
+envlist = py{27,34,35,36,37,py},py{27,37}-flake8,doclint,commitlint,docstrings
 minversion = 2.5.0
```

Comment:
If I remember correctly, `TOXENV=py38` didn't work until one of the more recent releases of tox. That makes me think that we'll need to bump this minimum version. Would you be willing to do that?

Reply:
Please also update this line.